docs/features/REST-sphubsitedata-type.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SPHubSiteData REST type
+description: SPHubSiteData REST type
+ms.date: 2/26/2018
+---
+
+# SPHubSiteData REST type
+
+> [!IMPORTANT]
+> The hub sites feature is currently in preview and is subject to change. It is not currently supported for use in production environments.
+
+Contains data describing a SharePoint Hub site.
+
+|Name     |Type   |Description |
+|---------|-------|------------|
+|themeKey |string |  |
```

Comment:
Could say "Identifies the theme currently applied to all associated sites."

Reply:
fixed